2023.5/staged/community/conda_build_config.yaml

```
@@ -5,25 +5,25 @@ hdf5:
 pandas:
 - '>=1.2.5'
 provenance_lib:
-- 2023.5.0
+- 2023.5.1
 python:
 - '3.8'
 q2_alignment:
 - 2023.5.0
 q2_assembly:
-- 2023.5.0.dev0+3.g8f369c5
+- 2023.5.0.dev0+4.g9c719cc
 q2_composition:
 - 2023.5.0
 q2_cutadapt:
-- 2023.5.0
+- 2023.5.1
 q2_dada2:
 - 2023.5.0
 q2_deblur:
 - 2023.5.0
 q2_demux:
 - 2023.5.0
 q2_diversity:
-- 2023.5.0
+- 2023.5.1
```

```suggestion
- 2023.5.0
```

Conda seems to be solving for a far older version of numba, which won't have the deprecation error I believe. Double check that this is true.